src/mysql_vm_helpers.py

```diff
@@ -830,7 +832,8 @@ def _run_mysqlcli_script(
                 if len(stdout) > 1 and "ERROR" in stdout[1]:
                     # some errors will not bubble up from spawned process
                     # but are reported in the stdout
-                    logger.error(stdout[1].strip())
+                    if log_errors:
```

Comment:
What's the error you wanted to show?

Reply:
it's actually an error that i did not want to log to the debug log (especially since cluster metadata will not exist before a unit joins the cluster - we will unnecessarily log errors)